bindings/Python/py11glue.cpp

```diff
@@ -899,16 +902,17 @@ PYBIND11_MODULE(adios2, m)
                      Single values will have a shape={1} numpy array
         )md")
 
-        .def("read",
-             (pybind11::array(adios2::py11::File::*)(
-                 const std::string &, const adios2::Dims &,
-                 const adios2::Dims &, const size_t)) &
-                 adios2::py11::File::Read,
-             pybind11::return_value_policy::take_ownership,
-             pybind11::arg("name"), pybind11::arg("start") = adios2::Dims(),
-             pybind11::arg("count") = adios2::Dims(),
-             pybind11::arg("block_id") = 0,
-             R"md(
+        .def(
+            "read",
+            (pybind11::array(adios2::py11::File::*)(
+                const std::string &, const adios2::Dims &, const adios2::Dims &,
+                const size_t, const std::string &order)) &
+                adios2::py11::File::Read,
+            pybind11::return_value_policy::take_ownership,
+            pybind11::arg("name"), pybind11::arg("start") = adios2::Dims(),
+            pybind11::arg("count") = adios2::Dims(),
+            pybind11::arg("block_id") = 0, pybind11::arg("order") = "",
```

Comment:
Should we make the default `K` explicit? It will help documenting this API.

Reply:
Well, I wanted to be the default to be changeable (via `adios2.File.read_order`), so that's why I didn't hardcode it there. If I put `"K"` there, I couldn't tell whether I get `"K"` because the user said `read(..., order='K')`, or just `read()`.

If you think we don't need a configurable default, yes, putting `"K"` there would be cleaner, and getting rid of all the `ReadMode` stuff. But in that case, there'd be no easy way to go back to current behavior, and since this introduces a backward-incompatible change in semantics, I figured there should at least be an easy way to go back to the old behavior.

Reply:
@germasch adding K would be cleaner to people used to numpy, other PRs already break the API for column-major generated data, so not a big a deal as long as there is clear documentation for the next release.